### PR TITLE
Fix History Chart Filter

### LINKED ISSUE #485: Add null tests in filter of history charts

### PR DESCRIPTION
**Pull request by Francisco2002**
Add null tests count

**How to test**
Compare [localhost](http://localhost:5173/tree/1bf329c696cf80000f1098bcdc23534e6fe14fe2?tableFilter=%7B%22buildsTable%22%3A%22all%22%2C%22bootsTable%22%3A%22all%22%2C%22testsTable%22%3A%22all%22%7D&origin=maestro&currentTreeDetailsTab=treeDetails.tests&diffFilter=%7B%22testStatus%22%3A%7B%22NULL%22%3Atrue%7D%7D&treeInfo=%7B%22gitBranch%22%3A%22for-kernelci%22%2C%22gitUrl%22%3A%22https%3A%2F%2Fgit.kernel.org%2Fpub%2Fscm%2Flinux%2Fkernel%2Fgit%2Farm64%2Flinux.git%22%2C%22treeName%22%3A%22arm64%22%2C%22commitName%22%3A%22v6.12-rc5-145-g1bf329c696cf%22%2C%22headCommitHash%22%3A%221bf329c696cf80000f1098bcdc23534e6fe14fe2%22%7D)  with [dashboard](https://dashboard.kernelci.org/tree/1bf329c696cf80000f1098bcdc23534e6fe14fe2?tableFilter=%7B%22buildsTable%22%3A%22all%22%2C%22bootsTable%22%3A%22all%22%2C%22testsTable%22%3A%22all%22%7D&origin=maestro&currentTreeDetailsTab=treeDetails.tests&diffFilter=%7B%22testStatus%22%3A%7B%22NULL%22%3Atrue%7D%7D&treeInfo=%7B%22gitBranch%22%3A%22for-kernelci%22%2C%22gitUrl%22%3A%22https%3A%2F%2Fgit.kernel.org%2Fpub%2Fscm%2Flinux%2Fkernel%2Fgit%2Farm64%2Flinux.git%22%2C%22treeName%22%3A%22arm64%22%2C%22commitName%22%3A%22v6.12-rc5-145-g1bf329c696cf%22%2C%22headCommitHash%22%3A%221bf329c696cf80000f1098bcdc23534e6fe14fe2%22%7D)

Close #485

### CODE REVIEW

File: backend/kernelCI_app/views/treeCommitsHistory.py [hunk 1]
```diff
@@ -207,7 +211,9 @@ def get(self, request, commit_hash):
                 SUM(CASE WHEN t.status = 'DONE' AND t.path LIKE 'boot%%' THEN 1 ELSE 0 END)
                 AS boots_done_count,
                 SUM(CASE WHEN t.status = 'SKIP' AND t.path LIKE 'boot%%' THEN 1 ELSE 0 END)
-                AS boots_skip_count
+                AS boots_skip_count,
+                SUM(CASE WHEN t.status IS NULL AND t.path LIKE 'boot%%' THEN 1 ELSE 0 END)
```
[the remark this answers is not in the document]
Comment (WilsonNet) [reply backend/kernelCI_app/views/treeCommitsHistory.py:215]: I know you didn't introduce this issue but you are already fixing the file

**Comment by WilsonNet** [comment]
nice job but for some reason it broke with null in this link
http://localhost:5173/tree/1bf329c696cf80000f1098bcdc23534e6fe14fe2?tableFilter=%7B%22buildsTable%22%3A%22all%22%2C%22bootsTable%22%3A%22inconclusive%22%2C%22testsTable%22%3A%22inconclusive%22%7D&origin=maestro&currentTreeDetailsTab=treeDetails.tests&diffFilter=%7B%22testStatus%22%3A%7B%22NULL%22%3Atrue%7D%7D&treeInfo=%7B%22gitBranch%22%3A%22for-kernelci%22%2C%22gitUrl%22%3A%22https%3A%2F%2Fgit.kernel.org%2Fpub%2Fscm%2Flinux%2Fkernel%2Fgit%2Farm64%2Flinux.git%22%2C%22treeName%22%3A%22arm64%22%2C%22commitName%22%3A%22v6.12-rc5-145-g1bf329c696cf%22%2C%22headCommitHash%22%3A%221bf329c696cf80000f1098bcdc23534e6fe14fe2%22%7D

**Review by WilsonNet — APPROVED** [review]
nice job!!

**Review by lfjnascimento — APPROVED** [review]
LGTM

**Pull request merged**